src/rpc/blockchain.cpp

```diff
@@ -1640,6 +1640,37 @@ static T CalculateTruncatedMedian(std::vector<T>& scores)
     }
 }
 
+static constexpr int NUM_GETBLOCKSTATS_PERCENTILES = 5;
```

Comment:
nit: I think it's better to place `constexpr`s at the top of the file (especially since this is used in multiple functions in this file)